modules/components/src/Aggs/BooleanAgg.js

```diff
@@ -0,0 +1,124 @@
+import React, { Component } from 'react';
+import AggsWrapper from './AggsWrapper.js';
+import { css } from 'emotion';
+import {
+  toggleSQON,
+  inCurrentSQON,
+  replaceSQON,
+  removeSQON,
+} from '../SQONView/utils';
+import './BooleanAgg.css';
+
+const booleanValues = {
+  true: 'true',
+  false: 'false',
+};
+
+export default ({
+  field,
+  buckets,
+  handleValueClick = () => {},
+  isTrueSelected = () => false,
```

Comment:
can this match the `isActive` api in the term facet? seems like it's doing the same thing